### PR TITLE
HAC-Hypershift

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -188,8 +188,8 @@ tests:
       TEST_TIMEOUT: "30"
     test:
     - ref: cucushift-helper-service-build-info
-    - ref: idp-htpasswd
```
Comment (kuiwang02) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:191]: @LiangquanLi930 please keep it. it is reserved for late usage. some golang cases will use these users.

**Comment by kuiwang02** [comment]
@LiangquanLi930 please check failure of https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/33328/rehearse-33328-periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-e2e-aws-ipi-ovn-hypershift-guest-p2/1583409433147346944

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml (hunk 1 above)
Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:191]: `idp-htpasswd` not support hyperhshift guest cluster

Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:191]: https://github.com/openshift/release/pull/33365

Comment (kuiwang02) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:191]: I do think #33365 has problem. will talk to you.

**Comment by LiangquanLi930** [comment]
> @LiangquanLi930 please check failure of https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/33328/rehearse-33328-periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-e2e-aws-ipi-ovn-hypershift-guest-p2/1583409433147346944

@kuiwang02  Hi, This failed because of the problem with the release image, I have submitted a https://github.com/openshift/release/pull/33347 to fix it

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml (hunk 1 above)
Comment (kuiwang02) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:191]: ok, got it. thanks

**Comment by kuiwang02** [comment]
> > @LiangquanLi930 please check failure of https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/33328/rehearse-33328-periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-e2e-aws-ipi-ovn-hypershift-guest-p2/1583409433147346944
> 
> @kuiwang02 Hi, This failed because of the problem with the release image, I have submitted a #33347 to fix it

ok, got it. thanks
let's wait it merged.

**Comment by LiangquanLi930** [comment]
@yapei @yanpzhan PTAL this PR, thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml (hunk 1 above)
Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:191]: https://github.com/openshift/release/pull/33365 merged

**Comment by kuiwang02** [comment]
@LiangquanLi930 the job still fails to create hosted cluster.

**Pull request closed without merging**